### ISSUE TITLE
Convolutional Neural Network

### ISSUE THREAD

**Issue by rshfsr** [nus-mmsys/tmf#21]
- 2 convolution layer of 7 Convolution (12 * 12 size)
- 2 max pool layer of 7 MaxPool with Relu (3 * 3)
- 2 fully connected layers with Relu
- 1 Softmax regression layer
